book/src/themes.md

```diff
@@ -296,6 +296,9 @@ These scopes are used for theming the editor interface:
 | `ui.statusline.insert`            | Statusline mode during insert mode ([only if `editor.color-modes` is enabled][editor-section]) |
 | `ui.statusline.select`            | Statusline mode during select mode ([only if `editor.color-modes` is enabled][editor-section]) |
 | `ui.statusline.separator`         | Separator character in statusline                                                              |
+| `ui.bufferline`                   | Style for the buffer line                                                                      |
+| `ui.bufferline.active`            | Style for active buffers in buffer line                                                        |
```

Comment:
Since there can only be one active at a time:

```suggestion
| `ui.bufferline.active`            | Style for the active buffer in buffer line                                                        |
```